charms/istio-pilot/src/charm.py

```diff
@@ -647,7 +664,7 @@ def _report_handled_errors(self, errors):
             )
             self._log_and_set_status(status_to_publish)
             for i, error in enumerate(errors):
-                self.log.info(f"Handled error {i}/{len(errors)}: {error.status}")
+                self.log.info(f"Handled error {i+1}/{len(errors)}: {error.status}")
```

Comment:
you can use `enumerate(errors, start=1)`
```suggestion
                self.log.info(f"Handled error {i}/{len(errors)}: {error.status}")
```